homeassistant/components/toon/climate.py

```diff
@@ -121,7 +123,8 @@ def set_temperature(self, **kwargs) -> None:
 
     def set_preset_mode(self, preset_mode: str) -> None:
         """Set new preset mode."""
-        self.toon.thermostat_state = preset_mode
+        if preset_mode is not None:
+            self.toon.thermostat_state = preset_mode
```

Comment:
log errer?

Reply:
Nope, ignoring, in this case, seems to be the best fit.
It is a selectable option in the frontend.